resources/views/components/input/select/region.blade.php

```diff
@@ -0,0 +1,21 @@
+@props(['country' => ''])
+<graphql
+    v-if="{{ $country }}"
+    query="query country($id: String) { country(id: $id) { available_regions { code id name } } }"
+    v-bind:variables="{ id: {{ $country }} }"
+    v-bind:cache="'regions_' + {{ $country }}"
+    v-bind:key="{{ $country }}"
+>
+    <div class="contents" slot-scope="{ data }">
+        <template v-if="data && data.country.available_regions">
+            <x-rapidez::input.select
+                {{ $attributes }}
+                v-bind:required="'{{ Rapidez::config('general/region/state_required', '') }}'.split(',').includes({{ $country }})"
```

Comment:
No need for defining a default when there isn't one?

```suggestion
                v-bind:required="'{{ Rapidez::config('general/region/state_required') }}'.split(',').includes({{ $country }})"
```

Reply:
I did this here just to make sure you don't get a stringified `null` in there, although I just checked and `null` becomes an empty string anyway, including when passed through the `e()` function.